src/main/resources/db/migration/V1_2__insert_prison_gp_practice.sql

```diff
@@ -53,7 +53,6 @@ INSERT INTO prison_gp_practice VALUES ('HMI', 'Y05528');
 INSERT INTO prison_gp_practice VALUES ('HCI', 'Y03223');
 INSERT INTO prison_gp_practice VALUES ('ISI', 'Y05765');
 INSERT INTO prison_gp_practice VALUES ('IWI', 'Y05532');
-INSERT INTO prison_gp_practice VALUES ('IWI', 'Y05534');
```

Comment:
duplicate row for `IWI` - temporarily take out until work out best way to resolve.